certificates/src/lib.rs

```diff
@@ -38,7 +39,7 @@ pub enum CertificateError {
     #[error("Error validating certificate hostname")]
     ErrorValidatingHostname(#[source] openssl::error::ErrorStack),
 
-    #[error("Certificate not valid for {hostname:?}: {cert_description:?}")]
+    #[error("Certificate not valid for {hostname:?}: {cert_description}")]
```

Comment:
Update error text to indicate possible set of hostnames?
```suggestion
    #[error("Certificate not valid for given hostnames ({hostname:?}): {cert_description}")]
```